components/Card/card.css

```diff
@@ -0,0 +1,78 @@
+@import "/docs.css";
+@import "components\Button\button.css";
+/* @import "/components/Utilities/utilities.css"; */
+
+.card{
+    display: flex;
+    flex-direction: column;
+    max-width:min-content;
+    border-radius: 10px;
+    position:relative;
+    margin: 0.5rem;
+    background-color: var(--white-color);
+}
+
+.card-horizontal{
+    display: flex;
+    flex-direction: row;
+    max-width:fit-content;
+    border-radius: 10px;
+    position:relative;
+    margin: 0.5rem;
+    background-color: var(--white-color);
+}
+
+.text-card{
+    width: 60ch;
+    max-width: fit-content;
+}
+
+.card-description{
+    flex-wrap: wrap;
+    max-width:40ch ;
+}
+
+.card-img{
+    border-radius: 10px;
```

Comment:
Next to card-img class you have already used br-md in line 75 in card.html. So you can remove this here